tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -90,22 +93,31 @@ private class DialogAccess {
 
 		Button incrementalCheckBox;
 
+		Button replaceAllButton;
+
 		Button regExCheckBox;
 
 		private Supplier<Shell> shellRetriever;
 
 		private Runnable closeOperation;
 
+		public Combo replaceCombo;
+
+		public Button selectedRangeRadioButton;
```

Comment:
I would propose to place the added fields at more reasonable positions, i.e., `replaceCombo` after `findCombo` without `public` modifier, `selectedRangeRadioButton` after `globalRadioButton` without public modifier, and `replaceAllButton` and `replaceAllButton` not in between the state-changing buttons.

Reply:
I have removed the test and added a similar test in FindReplaceLogicTest with this commit: https://github.com/eclipse-platform/eclipse.platform.ui/commit/fa4cfb1121b8ac0f2fe8f1bbb0ae46cc656dc22e